packages/x-charts/src/LineChart/extremums.ts

```diff
@@ -43,8 +43,9 @@ export const getExtremumY: ExtremumGetter<'line'> = (params) => {
         const { area, stackedData } = series[seriesId];
         const isArea = area !== undefined;
 
+        // Since this series is not used to display an area, we do not consider the base (the d[0]).
         const getValues: GetValuesTypes =
-          isArea && axis.scaleType !== 'log' ? (d) => d : (d) => [d[1], d[1]]; // Since this series is not used to display an area, we do not consider the base (the d[0]).
+          isArea && axis.scaleType !== 'log' ? (d) => d : (d) => [d[1], d[1]];
```

Comment:
```suggestion
          isArea && axis.scaleType !== 'log' && typeof series[seriesId].baseline !== 'string' ? (d) => d : (d) => [d[1], d[1]];
```

If baseline is min or max we don't want to use the 0 in the extremum computation